backend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "main": "src/index.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest --colors --runInBand",
```

Comment:
I see why other groups used `runInBand`, but why here? 
It seems to work without it and it's twice as fast.